### ISSUE TITLE
private modules section style is inconsistent

### ISSUE THREAD

**Issue by snopeks** [npm/docs#485]
Compared to the rest of the way the docs are laid out, the private modules section doc links are listed horizontally rather than vertically. Fixing this would make it look more unified with the other doc links. 

As well, the `02 - downloading modules to CI/deployment servers` link looks a little "off" divided into two lines. 
https://docs.npmjs.com/

It's still working as expected, so this isn't a huge issue :)

**Comment by PrototypeAlex** [comment]
TL;DR rabbit holes ahead

Had a decent go at this one for a bit of fun, but found without changing the column system into a grid system I could find no easy solution.

My attempts ended up walking the line of adding classes to the ULs, via javascript, based on the amount of children they have, because you can't do that in css sadly. 

Looked something like this:
```  
$('li:nth-child(1):nth-last-child(2)').parent().addClass('oneColumn')
$('li:nth-child(1):nth-last-child(3)').parent().addClass('twoColumns')
$('li:nth-child(1):nth-last-child(4)').parent().addClass('twoColumns')
```

Anyway, that worked, but when you drop the column count down in css, it spreads the width of the columns to fill the width of the ul, so with 2 columns they each take up 50%, and so on...

So long story short, it'd be nice to fix this, but would be a bit of work, unless there's a way that I can't see. :telescope:

**Comment by denmch** [comment]
This could be mitigated with `orphans: 2` if Firefox weren't a problem. But CSS columns seem problematic generally. How do folks feel about using ol' reliable jQuery? I'm sure there's a more elegant solution, but here's a quick hack that changes the column count to one if a UL has fewer than three children, thus preventing short lists from spanning multiple columns:

    $('li').each(function () {
      if($(this).parent().children('li').length < 3) {
        $(this).parent().css('column-count', 1);
      }
    });

Short of that it may be easiest to recode and move away from CSS columns.

**Comment by granteagon** [comment]
Submitted a pull request for this issue.  Here's what it looks like: http://cl.ly/253S2B2t0W3z

**Comment by granteagon** [comment]
@snopeks Have you had a chance to look at the PR?
